graph/typeDefs.graphql

```diff
@@ -1523,6 +1529,9 @@ type RootMutation {
 
   # forceScrapeAsset will force scrape the Asset with the given ID.
   forceScrapeAsset(id: ID!): ForceScrapeAssetResponse
+
+  # delUser will delete the user with the specified id.
+  delUser(id: ID!): DelUserResponse
```

Comment:
I prefer `deleteUser` and `DeleteUserResponse` instead of shortening it. Wdyt?

Reply:
`delete` is a reserved word in javascript :(